FreshRSS/FreshRSS
# Sticky buttons

Inspired by #6014
Pre-work done by #6202

Keep the submit button always reachable. 


have a look onto the logs
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/7f27b7a1-65ac-4906-8603-574b7c7ed54c)

have a look onto the feed configuration
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/2e005276-cf82-4f76-a1c8-ff8623f37817)



How to test the feature manually:

1. go to `display` option and enable the sticky buttons (last option)
2. see the sticky buttons at the bottom of window

Pull request checklist:
